shell/platform/embedder/embedder.h

```diff
@@ -522,6 +543,10 @@ typedef struct {
   /// `FlutterPresentInfo` struct that the embedder can use to release any
   /// resources. The return value indicates success of the present call.
   BoolPresentInfoCallback present_with_info;
+  /// Specifying this callback is a requirement for dirty region management. The
```

Comment:
`is a requirement for dirty region management` can be clarified a little more IMO. The description needs to briefly describe:

1. Why someone would want to use this over `fbo_with_frame_info_callback`.
2. Talk about what would happen if `fbo_callback` or `fbo_with_frame_info` callback is present along with this. Update the documentation for those methods as well.

Reply:
I just added a more thorough description of the callback. I also updated some of the other callback headers to explain how dirty region management might depend on them